.github/workflows/publish-embedded-packages.yaml

```diff
@@ -0,0 +1,228 @@
+
+      - name: Publish AAR
+        if: ${{ env.DRY_RUN == 'false' }}
+        # TODO: can we do some kind of dry run?
+        env:
+          EC_VERSION: ${{ env.ARTIFACT_VERSION }}
+          ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.MAVEN_RELEASE_USERNAME }}
+          ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.MAVEN_RELEASE_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.GPG_SIGNING_KEY }}
+          ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.GPG_SIGNING_KEY_PASSWORD }}
+        run: embedded/android/publish_android_package.sh -s
+
+      - name: Add release note
+        if: ${{ env.DRY_RUN == 'false' }}
+        uses: softprops/action-gh-release@v2
+        with:
+          append_body: true
+          body: |
+            ## Android AAR embedded package
+
+            This package provides a build of Element Call that can be used as an embedded widget within another application.
+
+            ```
+            dependencies {
+              implementation 'io.element.android:element-call-embedded:${{ env.ARTIFACT_VERSION}}'
+            }
+            ```
+
+  publish_ios:
+    needs: build_element_call
+    if: always()
+    name: Publish SwiftPM Library
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: element-call
+
+      - name: 📥 Download built element-call artifact
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          run-id: ${{ github.event.workflow_run.id || github.run_id }}
+          name: build-output-embedded
+          path: element-call/embedded/ios/Sources/dist
+
+      - name: Checkout element-call-swift
+        uses: actions/checkout@v4
+        with:
+          repository: element-hq/element-call-swift
+          path: element-call-swift
+          token: ${{ secrets.SWIFT_RELEASE_TOKEN }}
+
+      - name: Copy files
+        run: rsync -a --delete --exclude .git element-call/embedded/ios/ element-call-swift
+
+      - name: Get artifact version
+        run: echo "ARTIFACT_VERSION=${VERSION:1}" >> "$GITHUB_ENV"
+
+      - name: Commit and tag
+        working-directory: element-call-swift
+        run: |
+          git config --global user.email "ci@element.io"
+          git config --global user.name "Element CI"
+          git add -A
+          git commit -am "Release ${{ env.VERSION }}"
+          git tag -a ${{ env.ARTIFACT_VERSION }} -m "${{ github.event.release.html_url }}"
+
+      - name: Push
+        working-directory: element-call-swift
+        run: |
+          git push --tags ${{ env.DRY_RUN == 'true' && '--dry-run' || '' }}
+
+      - name: Add release note
+        if: ${{ env.DRY_RUN == 'false' }}
+        uses: softprops/action-gh-release@v2
```

Comment:
Should be pinned to avoid supply chain attack

Reply:
```suggestion
        uses: softprops/action-gh-release@c95fe1489396fe8a9eb87c0abf8aa5b2ef267fda # v2
```